cmd/ocsp-responder/main.go

```diff
@@ -55,34 +55,37 @@ serialNumber field, since we will always query on it.
 
 */
 type DBSource struct {
-	dbMap     *gorp.DbMap
+	dbMap     dbSelector
 	caKeyHash []byte
+	log       *blog.AuditLogger
+}
+
+type dbSelector interface {
```

Comment:
This needs a comment to clarify that we're wrapping gorp, not abandoning it.

```
// Since the only thing we use from gorp is the SelectOne method on the 
// gorp.DbMap object, we just define the interface an interface with that method
// instead of importing all of gorp.
```